examples/ct/handlers.go

```diff
@@ -174,19 +174,21 @@ type LogContext struct {
 		allRsps *expvar.Map // http.rc => expvar.Int  (as "http-all-rsps")
 		rsps    *expvar.Map // entrypoint => expvar.Map[http.rc => expvar.Int]  (as "http-rsps")
 	}
+	rejectExpired bool
 }
 
 // NewLogContext creates a new instance of LogContext.
-func NewLogContext(logID int64, prefix string, trustedRoots *PEMCertPool, rpcClient trillian.TrillianLogClient, signer *crypto.Signer, rpcDeadline time.Duration, timeSource util.TimeSource) *LogContext {
+func NewLogContext(logID int64, prefix string, trustedRoots *PEMCertPool, rpcClient trillian.TrillianLogClient, signer *crypto.Signer, rpcDeadline time.Duration, timeSource util.TimeSource, rejectExpired bool) *LogContext {
```

Comment:
Might be a smidgeon clearer if we shuffled `rejectExpired` to be just after `trustedRoots`, so the two validation-related arguments are together.